source/installation.txt

```diff
@@ -50,3 +50,4 @@ For instructions on installing Relational Migrator, see the following pages:
    /installation/install-on-a-single-machine/install-on-a-single-machine
    /installation/install-on-an-unattended-server/install-on-an-unattended-server
    /installation/file-location
+   /installation/atlas-log-in
```

Comment:
I think having this on the installation drawer looks a bit odd, the task is not installation focused, its more so an additional step to access query-converter. 

I think it should be either part on the [enable query converter task](https://preview-mongodbjocelynmendez1.gatsbyjs.io/docs-relational-migrator/DOCSP-34504/code-generation/query-converter/enable-query-converter/) page or in the projects drawer. This depends if the Atlas sign in will be required for additional features, we should also add a reference to the query converter content pointing to this task as a BYB.